composer.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "octalmage/wp-unit-test-runner",
+  "type": "library",
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/octalmage/wp-unit-test-api-client-php.git"
```

Comment:
Does this really need to be a separate library, or can we consolidate into this repo?

For simplicity's sake, it'd be better to keep everything together if we can.